src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    private boolean completed = false;
+    private final String title;
+
+    public Task(String title) {
+        this.title = title;
+    }
+
+    public void markAsComplete() {
+        this.completed = true;
+    }
+
+    public void markAsIncomplete() {
+        this.completed = false;
+    }
+    @Override
+    public String toString() {
+        return String.format("[" + this.convert() + "] " + title);
```

Comment:
instead of building the string using concatenation inside String.format, we can leverage the format placeholders directly.




```suggestion
        return String.format("[%s] %s", this.convert(), title);
```